hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -180,6 +210,8 @@ public void migrate() {
 			final var app = getMetadata().app();
 			app.workingState().updatePrimitiveChildrenFrom(this);
 		}
+		log.info("Migration completed.");
+		logStateChildrenSizes();
```

Comment:
@rjuang would we still need it here ? `internalInit()` is called either way right ?

Reply:
Yes. We want to log twice:
1. In InternalInit before migration.
2. After migrate (if it happens).

This is to ensure we can check the state before and after migration and know whether we copied all elements.